### ISSUE TITLE
[tasks] Specify AFSC database in get_connected()

### ISSUE THREAD

**Issue by mkapur-noaa** [afsc-gap-products/gapindex#45]
### Issue Description

Update the prompt text in `get_connected(`) to specify that folks should enter their password for the AFSC Oracle database. Stock assessors have access via separate creds to AKFIN.

### Tasklist

- [ ] Update the quoted text in lines 14 & 15 here: https://github.com/afsc-gap-products/gapindex/blob/62f050e4e16c645562a8c44bef66b18949bc96c0/R/get_connected.R#L14

**Comment by MargaretSiple-NOAA** [comment]
@zoyafuso-NOAA I will leave you the satisfaction of closing this issue. BUT this makes me think of a fun github trick I learned...  If you write "fixes # blah" (where blah is the issue number) in your commit message for the code that fixes the issue, when you push the change... it will automatically close the issue for you! I forget when I learned this but it's very fun, blog post about it [here](https://github.blog/2013-01-22-closing-issues-via-commit-messages/).
